docs/zz_generated.seed.yaml

```diff
@@ -337,6 +337,12 @@ spec:
   # Metering configures the metering tool on user clusters across the seed.
   metering:
     enabled: false
+    # Interval defines period for metering timestamps
+    interval: ""
+    # IntervalInDays defines period for metering timestamps. Overwrites Interval.
+    intervalInDays: 0
+    # Schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+    schedule: ""
```

Comment:
Please ensure that this generated YAML actually represents the default values. For now, until we maybe have something more fancy and automated based on the openapi schema, please manually extend https://github.com/kubermatic/kubermatic/blob/master/codegen/example-yaml/main.go#L173